src/System.Drawing.Common/src/System/Drawing/Internal/SystemColorTracker.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if FEATURE_SYSTEM_EVENTS
```

Comment:
When the entire file is #if, maybe it's cleaner to use a condition in the MSBuild project instead.

Reply:
Eh, it's fine for now. I guess it is consistent.

Reply:
I went ahead and remove this.